pkgs/development/python-modules/pymumble/default.nix

```diff
@@ -9,26 +9,20 @@
   pythonOlder,
 }:
 
-buildPythonPackage rec {
+buildPythonPackage {
   pname = "pymumble";
-  version = "1.6.1"; # Don't upgrade to 1.7, version was yanked
+  version = "unstable-2024-10-20";
   format = "setuptools";
```

Comment:
```suggestion
  pyproject = false;
```

Reply:
Done, thank you! I switched to `pyproject = true` since it falls back to setuptools.